string bi_agent
string visitor_id
string location
---
bool authorized
